bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java

```diff
@@ -152,6 +158,7 @@ public void initialize() {
         final String host = config.host;
         final Integer port = config.port;
         final Integer numZones = config.numZones;
+        this.isMps4 = config.mps4;
```

Comment:
Just a thought... Instead of having a configuration flag, why not just check if the port is 5006 and set isMps4 accordingly? It would make it easier for the user by auto-detecting the condition that requires the protocol change. The odds of the MPS4 port changing from 5006 in the future is non-existent.

Reply:
Sounds good..  Given legrand shutdown the product line - the chance that anything changes in the future is pretty low :-)